### DISCUSSION TITLE
Ping to device-mqtt

### DISCUSSION THREAD

**Discussion by Thinhhoagn0211** [edgexfoundry/community#119] [Q&A]
I have a docker-compose.yml file to run services in EdgeX, i have ran both device-rest and device-mqtt version 2.3.0, i have tried ping to device-rest and device-mqtt through IP of its, why only device-rest i can ping, not device-mqtt (i used image from EdgexFoundry). please answer this question for me.

**Comment by cloudxxx8** [comment]
Both of them should be able to ping.  You need to check the log message from device-mqtt.

**Reply by lenny-goodell** [reply]
Also, what error our you getting? What URL are you using?
